modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -1506,6 +1518,18 @@ Optional<Long> highestRevision(boolean addition) {
                     .map(Map.Entry::getKey);
         }
 
+        /**
+         * Returns the highest revision which is presented in the {@link ZoneState#topologyAugmentationMap()}.
+         *
+         * @return The highest revision which is presented in the {@link ZoneState#topologyAugmentationMap()}.
+         */
+        Optional<Long> highestRevision() {
+            return topologyAugmentationMap().entrySet()
```

Comment:
Since you don't use addition there's no longer need in entrySet(), keySet() should be enough.